src/common/contentPreference.ts

```diff
@@ -114,6 +112,7 @@ const followUser: FollowEntity = async ({ ctx, id, status }) => {
 
     const contentPreference = repository.create({
       userId: ctx.userId,
+      feedId: ctx.userId,
```

Comment:
by default feedId is userId, for custom feeds its gonna be actual id of the feed

Reply:
Yeah internally userid is just yet another feed, i like it 👍